src/output.cpp

```diff
@@ -11,7 +11,8 @@ bool is_quoted_type(const SizedType &ty)
 {
   return ty.IsKstackTy() || ty.IsUstackTy() || ty.IsKsymTy() || ty.IsUsymTy() ||
          ty.IsInetTy() || ty.IsUsernameTy() || ty.IsStringTy() ||
-         ty.IsBufferTy() || ty.IsProbeTy() || ty.IsCgroupPathTy();
+         ty.IsBufferTy() || ty.IsProbeTy() || ty.IsCgroupPathTy() ||
+         ty.IsStrerrorTy();
```

Comment:
Perhaps this could be changed to a switch/case where all the variants are explicitly listed
(without default). Then compiler would warn if new type is added without updating all sites.

Reply:
Good idea! https://github.com/iovisor/bpftrace/pull/2925